### ISSUE TITLE
Separate highlighting for inline external links

### ISSUE THREAD

**Issue by Benjamin-Lee** [siko1056/language-sphinx#6]
For example,

```
`link text <example.com>`_
```

would be a link to example.com. Would it be possible to get different highlighting for the `example.com` part and the remainder of the text?

**Comment by siko1056** [comment]
Is this issue closed by v0.1.9 for you?

**Comment by Benjamin-Lee** [comment]
Yep. Thanks!

**Comment by Benjamin-Lee** [comment]
Just noticed that when you move the link onto a new line, the highlighting breaks, even though it's valid rst:

<img width="750" alt="example" src="https://user-images.githubusercontent.com/13618205/42243134-2af737be-7ec5-11e8-85fd-e1b359fac26d.png">

**Comment by siko1056** [comment]
Yes, some rules struggle with this single line restriction. In the code the respective lines responsible for the link formatting are https://github.com/siko1056/language-sphinx/blob/master/grammars/sphinx.cson#L100

Maybe you find faster than me a way to define this grammar rule to span multiple lines?  This might get applied to other rules as well.

In the README.md I provided some information at the bottom how to start developing the grammar easily, if you are willing to work on this :wink:
